pkg/checkpoint/common/moduleconfig.go

```diff
@@ -1,25 +0,0 @@
-package common
```

Comment:
I get why this would make sense in common, but as noted before I think there is a benefit to having the same skeleton structure in all modules (even if it is perhaps an overkill for this one), and other modules with more logic that benefit from splitting the logic into multiple `internal/parts/subpackages` (like mempool or availability) must in fact separate `moduleConfig` into a different subpackage than the main file of the module to avoid a cyclic import. 

Same as my other comment, feel free to resolve after reading this comment.

Reply:
I see the point. In that case, let's create a package named `config` or `params` or something like that and use it consistently. But I suggest putting this in the cleanup backlog, as this is a minor issue and a matter of polishing.